jvm/src/main/scala/tastyquery/jdk/ClasspathLoaders.scala

```diff
@@ -0,0 +1,194 @@
+package tastyquery.jdk
+
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.io.InputStream
+import tastyquery.ast.Trees.Tree
+import java.util.jar.JarFile
+import org.apache.commons.io.FileUtils
+import java.io.File
+import scala.jdk.CollectionConverters.*
+import tastyquery.reader.classfiles.Classpaths.{Universe, universe, Classpath}
+import org.apache.commons.io.IOUtils
+import java.nio.file.FileSystems
+import tastyquery.ast.Names.{SimpleName, EmptyPackageName, termName}
+import scala.util.Using
+import scala.collection.mutable
+import java.nio.file.Path
+import java.util.jar.JarEntry
+import tastyquery.ast.Names
+import scala.reflect.NameTransformer
+
+object ClasspathLoaders {
+
+  enum FileKind {
+
+    case Class, Tasty
+
+    val ext: String = {
+      import scala.language.unsafeNulls
+      this.toString.toLowerCase
```

Comment:
Using `toLowerCase` like that is not safe wrt. systems configured in Turkish, Azeri or Lithuanian. You must pass in `(java.util.locale.ROOT_LOCALE)`. But that doesn't link on Scala.js by default. Prefer
```scala
val ext: String = this match
  case Class => "class"
  case Tasty => "tasty"
```
or something like that.